### DISCUSSION TITLE
Is there any possibility to make it run in background like everything?

### DISCUSSION THREAD

**Discussion by benyaminl** [cboxdoerfer/fsearch#433] [Q&A] [answered]
As title stated, I think it will be fast when it run in background, and put it on front end when press hotkey, is that possible? Or it's not available for now? as I seen there are no docs about it

**Comment by cboxdoerfer** [comment, marked as answer]
That's not possible yet, but it is planned for a future release.

**Reply by cboxdoerfer** [reply]
What you can however do is to use systemd timers or cron to update the database in the background.

**Reply by benyaminl** [reply]
@cboxdoerfer AH Yes, I check the cli switch, so it does help, and disabling start up rebuilding index does help, thanks! 

note for reader :
`fsearch -u` in crontab
